packages/components/src/globals/scss/_layer.scss

```diff
@@ -0,0 +1,93 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+//-------------------------------------------
+// 📑 Layer
+// ------------------------------------------
+//
+//   Layer                 ||  Elevation
+//   ==========================================
+//   0 - Base              ||  0
+//   1 - Flat              ||  1
+//   2 - Raised            ||  2
+//   3 - Overlay           ||  8
+//   4 - Pop-out           ||  24
+//   ==========================================
+//   Custom: Left Nav      ||  16
+//   Custom: Global Header ||  12
+//
+
+// Box shadow variables
+
+/// Box shadow color
+/// @access private
+/// @type Value
+/// @group global-layer
+/// @example box-shadow: 0px 3px 3px 0 $box-shadow;
+$box-shadow: rgba(0, 0, 0, 0.2);
+
+/// Raised box shadow
+/// @access private
+/// @type Value
+/// @group global-layer
+$box-shadow--raised: 0 1px 2px 0 $box-shadow;
+
+/// Overlay box shadow
+/// @access private
+/// @type Value
+/// @group global-layer
+$box-shadow--overlay: 0 4px 8px 0 $box-shadow;
```

Comment:
Should we change all of the spreads to be the same as our current box-shadow, e.g. `2px 6px`? That way no matter which layer they are using, it will output the proper spread.

```scss
$box-shadow--overlay: 0 2px 6px 0 $box-shadow;
$box-shadow--sticky-nav: 0 2px 6px 0 $box-shadow;
```

Reply:
@tw15egan sounds good to me! Would this be for all the `bx-shadow--*` variables?

Reply:
Yeah, that's what I had in mind. That way, it won't break for anyone, but we'll make sure they are using the correct values 👍